iroh-relay/src/dns.rs

```diff
@@ -548,4 +566,31 @@ pub(crate) mod tests {
         let result = stagger_call(f, &delays).await.unwrap();
         assert_eq!(result, 5)
     }
+
+    #[tokio::test]
+    #[traced_test]
+    async fn jitter_test_zero() {
+        let jittered_delay = add_jitter(&0);
+        assert_eq!(jittered_delay, Duration::from_secs(0));
+    }
+
+    //Sanity checks that I did the math right
+    #[test]
+    #[traced_test]
+    fn jitter_test_nonzero_positive() {
```

Comment:
Probably nitpicking.  But the naming of `_negative` & `_positive` confused me.  Maybe `_lower_bound` and `_upper_bound` are better?